teuthology/__init__.py

```diff
@@ -38,6 +48,8 @@
     format='%(asctime)s.%(msecs)03d %(levelname)s:%(name)s:%(message)s')
 log = logging.getLogger(__name__)
 
+log.info('teuthology version: %s', __version__)
```

Comment:
I like everything about this PR except for this line. Do we *really* want this output for every single invocation of any teuthology command?